app/src/main/res/values-de/strings.xml

```diff
@@ -781,6 +781,7 @@ Signatur: %5$s</string>
     <string name="loading_details">lädt</string>
     <string name="site_plans">Lagepläne</string>
     <string name="interactive_map">Interaktive Karte</string>
+    <string name="share_location">Ort teilen</string>
```

Comment:
This translation is never used. 
Is this a mistake?

Reply:
Yes, a mistake. It's a leftover from the first iteration where there was a button with text. (see screenshot in original pull request)